### PR TITLE
Add number line audio and replay button

### PR DESCRIPTION
**Pull request by zwatson2001**

### CODE REVIEW

File: task-launcher/src/tasks/math/trials/sliderStimulus.js [hunk 1]
```diff
@@ -3,12 +3,32 @@ import _shuffle from 'lodash/shuffle';
...
+    replayButton.innerHTML = replayButtonSvg;
+    replayButton.id = 'replay-btn-revisited';
+    replayButton.classList.add('replay'); 
+    replayButton.disabled = true;
+    contentWrapper.insertBefore(replayButton, prompt);
+  
+    const cue = responseType === 'button' ? 'numberLinePrompt1' : 'numberLineSliderPrompt1';
+  
+    const audioFile = mediaAssets.audio[cue] || '';
+    // Returns a promise of the AudioBuffer of the preloaded file path.
```
Comment (asengupta3) [inline task-launcher/src/tasks/math/trials/sliderStimulus.js:25]: I think this comment is not valid. This is not returning a promise.

File: task-launcher/src/tasks/math/trials/sliderStimulus.js [hunk 2]
```diff
@@ -3,12 +3,32 @@ import _shuffle from 'lodash/shuffle';
...
+    PageAudioHandler.playAudio(audioFile, () => {
+      // set up replay button audio after the first audio has played
+      setupReplayAudio(cue);
```
Comment (asengupta3) [inline task-launcher/src/tasks/math/trials/sliderStimulus.js:28]: Maybe check for `cue` to have length and only then call setupReplayAudio. So something like:
```
if (cue) {
   setupReplayAudio(cue);
}
```

File: task-launcher/src/tasks/math/trials/sliderStimulus.js [hunk 3]
```diff
@@ -208,6 +228,38 @@ export const slider = {
...
+          .getAudioBuffer(mediaAssets.audio[cue]);
+
+      audioSource = jsPsychAudioCtx.createBufferSource();
+      audioSource.buffer = audioBuffer;
+      audioSource.connect(jsPsychAudioCtx.destination);
+      audioSource.start(0);
+    }
+
+    playAudioCue();
+    */
```
Comment (asengupta3) [inline task-launcher/src/tasks/math/trials/sliderStimulus.js:262]: Please remove the commented code.

File: task-launcher/src/tasks/math/trials/sliderStimulus.js (hunk 2 above)
Comment (zwatson2001) [reply task-launcher/src/tasks/math/trials/sliderStimulus.js]: I'll make that change, thanks!

**Review by kachergis — APPROVED** [review]
looks good -- thank you!

**Pull request merged**